typescript/sdk/src/token/types.ts

```diff
@@ -58,22 +58,25 @@ const xERC20LimitConfigSchema = z.object({
 });
 export type XERC20LimitConfig = z.infer<typeof xERC20LimitConfigSchema>;
 
+const xERC20ExtraLockboxLimitConfigsSchema = z.object({
```

Comment:
It just occurs to me that there is really no reason why we should call this extra lockboxes, from the perspective of the xerc20 token, they are just other bridges, wdyt of us renaming this?